pennylane/ops/qubit/qchem_ops.py

```diff
@@ -562,8 +592,7 @@ class OrbitalRotation(Operation):
     """
     num_wires = 4
     num_params = 1
-    grad_method = "A"
-    grad_recipe = four_term_grad_recipe
+    grad_method = "F"
```

Comment:
```suggestion
    grad_method = None
```
this should force the decomposition

Reply:
Ah yes of course! 
I'll skip this one and use the eight-term rule above instead?